server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -52,16 +52,31 @@ export function create(
 	const enforceServerGeneratedDocumentId: boolean =
 		config.get("alfred:enforceServerGeneratedDocumentId") ?? false;
 
+    // Throttling logic for per-tenant rate-limiting at the HTTP route level
 	const tenantThrottleOptions: Partial<IThrottleMiddlewareOptions> = {
 		throttleIdPrefix: (req) => getParam(req.params, "tenantId") || appTenants[0].id,
 		throttleIdSuffix: Constants.alfredRestThrottleIdSuffix,
 	};
+    const tenantThrottler = tenantThrottlers.get(Constants.generalRestCallThrottleIdPrefix);
 
-	// Throttling logic for creating documents to provide per-cluster rate-limiting at the HTTP route level
-	const createDocThrottleOptions: Partial<IThrottleMiddlewareOptions> = {
+    const createDocTenantThrottleOptions: Partial<IThrottleMiddlewareOptions> = {
+		throttleIdPrefix: (req) => getParam(req.params, "tenantId") || appTenants[0].id,
+		throttleIdSuffix: Constants.createDocThrottleIdPrefix,
+	};
+    const getSessionTenantThrottleOptions: Partial<IThrottleMiddlewareOptions> = {
+		throttleIdPrefix: (req) => getParam(req.params, "tenantId") || appTenants[0].id,
+		throttleIdSuffix: Constants.alfredRestThrottleIdSuffix,
```

Comment:
throttleIdSuffix: Constants.getSessionSuffix?

similar change for the below throttleOptions?

Reply:
Made the change. :)

Could I know which line do you mean for below throttleOptions suffix? The below one should be correct

Reply:
line number 78, throttleIdSuffix for getSessionClusterThrottleOptions should also be getSession and not alfredRestThrottleIdSuffix right?

Reply:
oh wait, because its per cluster, the key will be getsession_<alfred rest something> ?

Reply:
Yes, the key would be getsession_AlfredRest  for the get session in per cluster level.